mw-webapp/src/logic/wayPage/reportsTable/WayColumns.tsx

```diff
@@ -39,6 +39,11 @@ interface RenderModalContentParams {
    */
   description: string;
 
+  /**
+   * Name of element (jobDone, planForTomorrow, way, etc)
+   */
+  text: string;
```

Comment:
Why just element name? What if I want to render other message?

Reply:
Deleted text param and used description for render any text inside confirm